modules/control/common/trajectory_analyzer.cc

```diff
@@ -229,5 +232,37 @@ PathPoint TrajectoryAnalyzer::FindMinDistancePoint(const TrajectoryPoint &p0,
   return p;
 }
 
+void TrajectoryAnalyzer::TrajectoryTransformToCOM(
+    const double rear_to_com_distance) {
+  CHECK_GT(trajectory_points_.size(), 0);
+  for (size_t i = 1; i < trajectory_points_.size(); ++i) {
```

Comment:
Comment why we start with 1 while not 0.

Reply:
Thanks for the reminder. This is due to a mis-copy from other function; in this function the logic should cover from 0 to end.